attendee-data-editor.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * Plugin Name:       Event Tickets Plus Extension: Attendee Data Editor
+ * Plugin URI:        https://theeventscalendar.com/extensions/attendee-data-editor/
+ * GitHub Plugin URI: https://github.com/mt-support/tribe-ext-attendee-data-editor
+ * Description:       Makes it possible and easy for admins to edit custom attendee meta fields.
+ * Version:           1.2.1
+ * Extension Class:   Tribe__Extension__Attendee_Data_Editor
+ * Author:            Modern Tribe, Inc.
+ * Author URI:        http://m.tri.be/1971
+ * License:           GPL version 3 or any later version
+ * License URI:       https://www.gnu.org/licenses/gpl-3.0.html
+ * Text Domain:       match-the-plugin-directory-name
+ *
+ *     This plugin is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     any later version.
+ *
+ *     This plugin is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *     GNU General Public License for more details.
+ */
+
+// Do not load unless Tribe Common is fully loaded and our class does not yet exist.
+if (
+	class_exists( 'Tribe__Extension' )
+	&& ! class_exists( 'Tribe__Extension__Example' )
+) {
+	class Tribe__Extension__Attendee_Data_Editor extends Tribe__Extension {
+		const REQUIRED_PHP_VERSION = '5.6';
+
+		private $dir = '';
+		private $url = '';
+
+		/**
+		 * Register plugin requirements (ET+ 4.7 or greater) with the extension manager.
+		 */
+		public function construct() {
+			$this->add_required_plugin( 'Tribe__Tickets_Plus__Main', '4.7.0' );
+		}
+
+		/**
+		 * Perform sanity checks and initial setup.
+		 */
+		public function init() {
+			if ( ! $this->php_version_check() ) {
+				return;
+			}
+
+			$this->dir = __DIR__;
+			$this->url = plugin_dir_url( __FILE__ );
+
+			load_plugin_textdomain( 'tribe-ext-attendee-data-editor', false, basename( __DIR__ ) . '/languages/' );
+			$this->class_loader();
+
+			// Avoid syntax errors if this file is loaded under PHP 5.2 or earlier
```

Comment:
But it _won't_ load unless PHP 5.6+

Reply:
What stops it from loading under PHP 5.5 or earlier? I didn't think the _Requires PHP_ tag, currently, was enforced?

Reply:
It never hits that because of failing PHP version check `return;`

I removed this bit and tested on PHP 5.3.29 and no debug error, just the admin notice, as expected.